modules/admin-ui-frontend/bower.json

```diff
@@ -31,5 +31,8 @@
   "moduleName": "adminNg",
   "resolutions": {
     "angular": "1.8.2"
-  }
+  },
+"resolutions": {
+  "angular-translate": "2.18.3"
+}
 }
```

Comment:
I suspect this is an unwanted change, is it?

Reply:
True, I'll remove it.